### ISSUE TITLE
Shared Screen Annotations Menu does not appear in Zoom.us

### ISSUE THREAD

**Issue by TC68** [pop-os/shell#444]
**Distribution (run `cat /etc/os-release`):**
 
NAME="Pop!_OS"
VERSION="20.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 20.04 LTS"
VERSION_ID="20.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
LOGO=distributor-logo-pop-os

 
**Related Application and/or Package Version (run `apt policy pop-shell` or provide the latest commit if building locally):**

pop-shell:
  Installed: 0.1.0~1593273798~20.04~9e8ab29
  Candidate: 0.1.0~1593273798~20.04~9e8ab29
  Version table:
 *** 0.1.0~1593273798~20.04~9e8ab29 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu focal/main amd64 Packages
       1001 http://ppa.launchpad.net/system76/pop/ubuntu focal/main i386 Packages
        100 /var/lib/dpkg/status

**Issue/Bug Description:**

When sharing a screen, window or whiteboard in a Zoom meeting, the annotations menu does not appear, so it is not possible to manage the annotations or change the annotation type.

**Steps to reproduce (if you know):**
Run Zoom
Start a new meeting and wait until connected
Share Screen > Whiteboard (or any other component)
Whiteboard appears and cursor changes to pen symbol. Can annotate the whiteboard using the pen, but can not use any of the other annotation features and the annotation tools toolbar does not appear.

**Expected behaviour:**
When sharing a screen/window/whiteboard an annotation tools toolbar appears allowing management of the annotations and a range of cursors, colours, text etc.

**Other Notes:**
Raised on reddit: https://www.reddit.com/r/pop_os/comments/hifnhe/using_zoom/

Reinstalling from the deb downloaded from zoom instead of pop_shop flatpak made no difference.

I do, however, also have cinnamon desktop installed on my computer. When I logged out of pop and logged in using cinnamon, zoom worked correctly with the annotations bar appearing as expected.

Disabling window tiling and pop_shell within the pop login session did not help.

**Comment by virtuosoj** [comment]
I am experiencing the same exact problem!    Disabling all extensions, restarting computer, reinstalling Zoom as .deb, .flatpak, .snap, none of these issues fixed the problem on Pop OS.    However, this problem was not present after installing Zoom flatpak on Linux Mint running off a liveusb!    This problem seems to be specific to Pop OS - which is sad because this is my preferred OS and Zoom is a necessary tool that I must use for my job.

**Comment by jmmaranan** [comment]
@TC68 - do you mind attaching a screenshot of what you are seeing. A screenshot of what should appear. (You don't need to paste the whole desktop)

**Comment by TC68** [comment]
![Zoom under cinnamon](https://user-images.githubusercontent.com/8788393/86853933-d66c4c00-c0fa-11ea-9f63-0d4bb09e4cb2.png)
![Zoom under pop](https://user-images.githubusercontent.com/8788393/86853937-d79d7900-c0fa-11ea-93ec-5f12a074a997.png)
Two screenshots. Both show me sharing a zoom whiteboard on the same computer. The first is pop 20.04 with a cinnamon window manager, the second is pop 20.04 with the current pop/gnome. In the cinnamon screenshot there is a black toolbar apparent which is not accessible in the gnome version.
The cinnamon behaviour is the expected behaviour.

**Comment by jmmaranan** [comment]
@TC68 - I am on pop session and it is showing for me. Do you mind checking other extensions?

Tiling enabled
![image](https://user-images.githubusercontent.com/348125/86855744-f8c58180-c088-11ea-8cb1-4b5332ab8cb4.png)

**Comment by TC68** [comment]
I disabled all extensions except 
Extensions
Pop Battery Icon Fix
Pop shell
Pop shop details
System76 power

Restarted the computer. And still have the same problem

**Comment by TC68** [comment]
Update
I normally use dual monitors - the one integrated into the laptop and an external monitor (dell, connected via USBC from my NVIDIA card).
If I unplug the second monitor, the problem does not occur in the pop session, even with all my usual gnome extensions running.

The mint session works fine with the second monitor

**Comment by jmmaranan** [comment]
@TC68 - do you have the `gnome-classic` in your xsessions/login options? Did you try on that too? Sorry if you meant current gnome/pop was for both - then disregard :)

One thing too is that, the other big tool bar above the Stop Share has an Icon for Whiteboard, did you try clicking that explicitly too? I am not sure how else I can help

**Comment by TC68** [comment]
Yes - I tried the whiteboard icon, but it didn't trigger the toolbar. In cinnamon, and in the current setup but without the external monitor, the toolbar appears automatically.
I have the default gnome/pop desktop environment (not sure how to refer to pop's version of the gnome desktop...) and the cinnamon desktop environment installed, but no others. I don't have gnome-classic.

**Comment by jmmaranan** [comment]
@TC68 - well I guess when you are going to use that feature for your zoom sessions, unplug for now until you get your message across your audience. It could be different resolutions or frequency.

**Comment by TC68** [comment]
Still seems weird that it happens under one desktop environment and not the other, but thanks for looking into it
